cloudconfig/util_test.go

```diff
@@ -0,0 +1,162 @@
+package cloudconfig
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cloudbase/garm-provider-common/params"
+	"github.com/google/go-github/v55/github"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	newCloudCfg     = NewDefaultCloudInitConfig()
+	bootstrapParams = params.BootstrapInstance{
+		ExtraSpecs: []byte(`{"runner_install_template": "test"}`),
```

Comment:
The value of the `runner_install_template` is a `[]byte{}` which in JSON form is a base64 encoded string.